scripts/pylint_extensions.py

```diff
@@ -904,31 +904,35 @@ def process_module(self, node):
 
 
 class SingleSpaceAfterYieldChecker(checkers.BaseChecker):
-  """Checks if only one space is used after a yield statement."""
```

Comment:
I indented this by 4 spaces instead of 2 as 4 spaces is our current coding style